src/nucleus/jdg.ml

```diff
@@ -669,7 +673,7 @@ let congr_lambda ~loc (EqTy (ctxa, ta1, ta2))
                  (EqTy (ctxb, b1, b2))
                  (EqTerm (ctxe, e1, e2, ty_e)) =
   if not (TT.alpha_equal_ty b1 ty_e)
-  then error ~loc (RuleInputMismatch ("congr-lambda", b1, ty_e))
+  then error ~loc (RuleInputMismatch ("congr-lambda", b1, "???", ty_e, "???"))
```

Comment:
If someone is using `congr_lambda` directly then they should know about it. If they're using it through a library they should report to the library people.